ee/agent/storage/stores.go

```diff
@@ -18,6 +18,7 @@ const (
 	ServerProvidedDataStore     Store = "server_provided_data"     // The store used for pushing values from server-backed tables.
 	TokenStore                  Store = "token_store"              // The store used for holding bearer auth tokens, e.g. the ones used to authenticate with the observability ingest server.
 	ControlServerActionsStore   Store = "action_store"             // The store used for storing actions sent by control server.
+	UpTimeHistoryStore          Store = "uptime_history"           // The store used for storing uptime history.
```

Comment:
Maybe we'd want something like `LauncherHistoryStore Store = "launcher_history"` -- I think that might be a little more extensible for the future, and would match `osquery_instance_history`. I don't feel strongly about it, though -- what do others think?

Reply:
I like that more too, it makes sense to me to report as just `uptime` for the table column but internally here it is a little more confusing what this refers to

Reply:
That makes sense, I updated the name to LauncherHistoryStore